### DISCUSSION TITLE
How does cache work and why does it break on Windows 10?

### DISCUSSION THREAD

**Discussion by sunny-lan** [stepmania/stepmania#2119] [Q&A]
https://github.com/stepmania/stepmania/wiki/Install-Guide#windows

This page says I need to exclude the Stepmania cache folder from Defender. Why? Does anyone know the root cause?

Edit: Stepmania 5.1

**Comment by yahooeny** [comment]
There's nothing that cache does that would be considered malicious, there's just performance issues inherent to how Defender scans for threats in real time.

Essentially, when the cache is built for the first time, Defender will then scan every single cache file. This as you could gather, thrashes i/o and makes cache building crawl to a halt.
You don't *have* to do this, in fact I've never put stepmania's cache in my exemptions for Defender.
